src/client.ts

```diff
@@ -799,7 +799,8 @@ type RoomEvents = RoomEvent.Name
 
 type RoomStateEvents = RoomStateEvent.Events
     | RoomStateEvent.Members
-    | RoomStateEvent.NewMember;
+    | RoomStateEvent.NewMember
+    | RoomStateEvent.Update;
```

Comment:
stylistically questionable, but this avoids future cases of having to juggle the semicolon:

```suggestion
    | RoomStateEvent.Update
    ;
```